### PR TITLE
DocumentViewController contraints.

### PR DESCRIPTION
**Pull request by Metaxa007**

### CODE REVIEW

File: OpenDocumentReader/DocumentViewController.swift [hunk 1]
```diff
@@ -62,7 +63,11 @@ class DocumentViewController: UIViewController, DocumentDelegate, GADBannerViewD
 
         searchBar.delegate = self
         searchBar.showsCancelButton = true
+        
+        searchBarHeightWhenShown = searchBar.heightAnchor.constraint(equalToConstant: 56)
+        searchBarHeightWhenHidden = searchBar.heightAnchor.constraint(equalToConstant: 0)
 
+        setVCcontraints()
```
Comment (TomTasche) [inline OpenDocumentReader/DocumentViewController.swift:70]:
```suggestion
        setVCconstraints()
```

File: OpenDocumentReader/DocumentViewController.swift [hunk 2]
```diff
@@ -83,6 +88,32 @@ class DocumentViewController: UIViewController, DocumentDelegate, GADBannerViewD
         loadBannerAd()
     }
     
+    func setVCcontraints() {
```
Comment (TomTasche) [inline OpenDocumentReader/DocumentViewController.swift:91]: can't we configure this via the Xcode UI?

**Comment by TomTasche** [comment]
works well (also with ads), thanks for the fix!

File: OpenDocumentReader/DocumentViewController.swift (hunk 2 above)
Comment (Metaxa007) [reply OpenDocumentReader/DocumentViewController.swift:91]: Of course it is possible, but more complicated to implement any changes in the future, if some problem like this occurs. Interface Builder has too many different options in different places. I would say, constraints via code are much easier to understand.

Comment (Metaxa007) [reply OpenDocumentReader/DocumentViewController.swift:91]: But let me know if you still want to have all the constraints via IB.

**Pull request merged**